modules/search/instant-search/lib/constants.js

```diff
@@ -10,6 +10,7 @@ export const RESULT_FORMAT_EXPANDED = 'expanded';
 export const RESULT_FORMAT_MINIMAL = 'minimal';
 export const RESULT_FORMAT_PRODUCT = 'product';
 export const MINUTE_IN_MILLISECONDS = 60 * 1000;
+export const DEFAULT_SORT = 'relevance';
```

Comment:
This should be `DEFAULT_SORT_KEY` I believe 🙂

Reply:
Fixed! Can't believe I forgot to press save on the file...